src/components/NavSections/RightMenu.js

```diff
@@ -1,14 +1,15 @@
 import React, { useState } from "react";
 import { Menu, Space, Badge, Modal, Col, Row, Typography, Image,message } from "antd";
 import Cookie from "js-cookie";
-
+import {withRouter} from 'react-router-dom'
 import {
   MailOutlined,
   ShoppingCartOutlined,
   UnorderedListOutlined,
   UserOutlined,
 } from "@ant-design/icons";
 import { useSelector } from "react-redux";
+import {Link} from 'react-router-dom'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7284710)
```suggestion
import {Link} from "react-router-dom"
```